src/@next/Banner/Banner.tsx

```diff
@@ -53,43 +64,87 @@ export const Banner = React.forwardRef<HTMLDivElement, BannerProps>(
       ? iconName
       : iconByStatus || iconNameStatusMap['info'];
 
+    if (type === 'static')
+      return (
+        <StyledBanner
+          ref={ref}
+          data-titled={!!title}
+          data-status={status}
+          {...props}
+        >
+          <StyledBannerTitleContainer>
+            <Icon name={iconNameValue} />
+            {title && (
+              <StyledBannerTitle>
+                <Typography as="span" variant="subtitle1">
+                  {title}
+                </Typography>
+              </StyledBannerTitle>
+            )}
+            {dismissable && (
+              <StyledCloseIconWrapper
+                role="button"
+                onClick={() => onDismiss?.()}
+              >
+                <Icon name="ri-close" />
+              </StyledCloseIconWrapper>
+            )}
+          </StyledBannerTitleContainer>
+          <StyledBannerContentContainer>
+            <Typography as="div" variant="body1">
+              {children}
+            </Typography>
+          </StyledBannerContentContainer>
+          <StyledBannerContentContainer>
+            {action &&
+              (secondaryAction !== 'undefined' ? (
+                <ButtonGroup>
+                  {action}
+                  {secondaryAction}
+                </ButtonGroup>
+              ) : (
+                { action }
+              ))}
+          </StyledBannerContentContainer>
+        </StyledBanner>
+      );
     return (
-      <StyledBanner
+      <StyledFixedBanner
         ref={ref}
-        data-titled={!!title}
         data-status={status}
+        data-nobutton={action ? '' : 'true'}
         {...props}
       >
-        <StyledBannerTitleContainer>
-          <Icon name={iconNameValue} />
-          {title && (
-            <StyledBannerTitle>
-              <Typography as="span" variant="subtitle1">
-                {title}
-              </Typography>
-            </StyledBannerTitle>
-          )}
-          {dismissable && (
-            <StyledCloseIconWrapper role="button" onClick={() => onDismiss?.()}>
-              <Icon name="ri-close" />
-            </StyledCloseIconWrapper>
-          )}
-        </StyledBannerTitleContainer>
-        <StyledBannerContentContainer>
+        {showIcon && (
+          <StyledFixedIconWrapper data-status={status}>
+            <Icon name={iconNameValue} />
+          </StyledFixedIconWrapper>
+        )}
+        <StyledFixedBannerContentContainer data-noicon={showIcon ? '' : 'true'}>
           <Typography as="div" variant="body1">
             {children}
           </Typography>
-        </StyledBannerContentContainer>
-        <StyledBannerContentContainer>
-          {action ||
-            (secondaryAction && (
+        </StyledFixedBannerContentContainer>
+        <StyledFixedBannerButtonContainer data-noicon={showIcon ? '' : 'true'}>
+          {action &&
```

Comment:
same, is this not owring? https://github.com/glints-dev/glints-aries/pull/838#discussion_r1209072975

Reply:
same reason as above